lib/model.js

```diff
@@ -3143,7 +3143,7 @@ class Model {
       }
       return values;
     }
-    return this.dataValues;
+    return Utils._.clone(this.dataValues);
```

Comment:
No, this will break things, you should be able to change datavalues and hit update and expect fields to be changed.

May be clone output of `toJSON`